obspy/io/seiscomp/inventory.py

```diff
@@ -446,12 +457,23 @@ def _read_channel(instrumentation_register, cha_element, _ns):
         if len(response_elements) == 0:
             msg = ("Could not find response tag with public ID "
                    "'{}'.".format(response_id))
-            raise obspy.ObsPyException(msg)
+            #raise obspy.ObsPyException(msg)
+            print ('{} - Response ERROR:'.format(seed_id))
+            print (msg)
+            print (' -> Omitting response information from Inventory')
+            print ('')
```

Comment:
This needs to be changed into a `warning` being issued rather than printing to stdout.

Reply:
Fixed it.